extension/intents/tabs/tabs.js

```diff
@@ -556,3 +556,23 @@ if (!buildSettings.android) {
     },
   });
 }
+
+intentRunner.registerIntent({
+  name: "tabs.countTabs",
+  async run(context) {
+    context.keepPopup();
+    const tabs = await browser.tabs.query({ currentWindow: true });
+    const card = {
+      answer: {
+        num: `${tabs.length}`,
+        text: "Open tabs",
+        eduText: "Click mic and say 'gather all Google tabs'",
+      },
+    };
+    await browser.runtime.sendMessage({
+      type: "showSearchResults",
```

Comment:
Hmm... we should probably make a followup to rename this, I can see why it would be the easy thing to work with here, but "showSearchResults" also isn't the right name anymore.

Reply:
I was actually in the middle of creating a new one for (what is now) AnswerCard, but I saw a note saying this issue should be a straightforward update, so I thought I was going deep down a rabbithole. If it's decided that it will be renamed, I would like to give it another shot!